public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```

Comment:
- [ ] Kindly remove all the boilerplates created by the CRA (texts, images, styles) from your `src `and `public `folders because you don't use them.